R/adducts.R

```diff
@@ -0,0 +1,449 @@
+#' @title Calculate mass-to-charge ratio
+#'
+#' @description
+#'
+#' `mass2mz` calculates the m/z value from a neutral mass and an adduct 
+#'      definition.
+#'
+#' @param x `numeric` neutral mass for which the adduct m/z shall be calculated.
+#'
+#' @param adduct `character` Valid adduct definition.
```

Comment:
Would be good to say what the *valid*/supported adduct definitions are. Something like "valid adduct definition; supported values are returned by [adductNames()]".